hack/tools/release/notes.go

```diff
@@ -403,8 +446,82 @@ REPLACE ME: A couple sentences describing the deprecation, including links to do
 
 	fmt.Println("")
 	fmt.Println("_Thanks to all our contributors!_ 😊")
+}
 
-	return 0
+func parseWeeklyUpdate(commits []*commit, merges map[string][]string, commitRange string, lastDay string) {
+	for _, c := range commits {
+		body := trimTitle(c.body)
+		var key, prNumber, fork string
+		switch {
+		case strings.HasPrefix(body, ":sparkles:"), strings.HasPrefix(body, "✨"):
+			key = features
+			body = strings.TrimPrefix(body, ":sparkles:")
+			body = strings.TrimPrefix(body, "✨")
+		case strings.HasPrefix(body, ":bug:"), strings.HasPrefix(body, "🐛"):
+			key = bugs
+			body = strings.TrimPrefix(body, ":bug:")
+			body = strings.TrimPrefix(body, "🐛")
+		case strings.HasPrefix(body, ":book:"), strings.HasPrefix(body, "📖"):
+			key = documentation
+			body = strings.TrimPrefix(body, ":book:")
+			body = strings.TrimPrefix(body, "📖")
+			if strings.Contains(body, "CAEP") || strings.Contains(body, "proposal") {
+				key = proposals
+			}
+		case strings.HasPrefix(body, ":seedling:"), strings.HasPrefix(body, "🌱"):
+			key = other
+			body = strings.TrimPrefix(body, ":seedling:")
+			body = strings.TrimPrefix(body, "🌱")
+		case strings.HasPrefix(body, ":warning:"), strings.HasPrefix(body, "⚠️"):
+			key = warning
+			body = strings.TrimPrefix(body, ":warning:")
+			body = strings.TrimPrefix(body, "⚠️")
+		default:
+			key = unknown
+		}
+
+		body = strings.TrimSpace(body)
+		if body == "" {
+			continue
+		}
+		body = fmt.Sprintf("\t - %s", body)
+		_, _ = fmt.Sscanf(c.merge, "Merge pull request %s from %s", &prNumber, &fork)
+		if key == documentation {
+			merges[key] = append(merges[key], prNumber)
+			continue
+		}
+		merges[key] = append(merges[key], formatMerge(body, prNumber))
+	}
+
+	// TODO Turn this into a link (requires knowing the project name + organization)
+	fmt.Println("Weekly update :rotating_light:")
+	fmt.Printf("Changes from %v a total of %d new commits where merged into main.\n\n", commitRange, len(commits))
```

Comment:
```suggestion
	fmt.Printf("Changes from %v a total of %d new commits were merged into main.\n\n", commitRange, len(commits))
```

Also, should we replace "main" with the `milestone` variable or otherwise detect which branch we're on?